spec/models/post_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  before(:each) do
+    @user = User.create(name: 'User', photo: 'image.png', bio: 'Teacher from Mexico.', posts_counter: 0)
+    @post = Post.create(author: @user, title: 'Test', text: 'This is a test post', comments_counter: 0,
+                        likes_counter: 0)
+  end
+
+  context 'when creating a Post' do
+    it 'should be valid with valid attributes' do
+      expect(@post).to be_valid
+      expect(@post.author).to eq(@user)
+    end
+
+    it 'should not be valid without a title' do
+      @post.title = nil
+      expect(@post).to_not be_valid
+    end
+
+    it 'should not be valid with the wrong type in counter' do
+      @post.comments_counter = 'zero'
+      expect(@post).to_not be_valid
+    end
+
+    it 'should not be valid with negative numbers' do
+      @post.comments_counter = -2
+      expect(@post).to_not be_valid
+    end
+  end
+
```

Comment:
- [ ] Great work on having tests for the comments_counter but it would be nice if you have tests for likes_counter because I don't find any tests for likes_counter 😨 Kindly make tests for `likes_counter` to make your milestone fulfill all the project requirements 👍